database/src/test/java/org/akvo/flow/database/DatabaseHelperTest.java

```diff
@@ -73,7 +80,7 @@ public void onUpgradeShouldUpgradeCorrectlyIfVersionResponseMigrate() {
     @Test
     public void onUpgradeShouldUpgradeCorrectlyIfVersionTransmissionMigrate() {
```

Comment:
Do we want to keep these old tests? in that case maybe its an idea to create a new constructor that adds in the question group table rather than using the old one?

Reply:
these tests test what should be upgraded depending on which version we are upgrading from, ie from version 1 to 2 there is only one upgrade done but 1 to 5 there are 5 of them